config/router_config_test.go

```diff
@@ -55,14 +55,6 @@ func TestString(t *testing.T) {
 	assert.Equal(t, n2[1], "")
 }
 
-func TestRouterInit(t *testing.T) {
-	//err := RouterInit(testVirtualServiceYML, testDestinationRuleYML)
-	//assert.NoError(t, err)
-	//
-	//err = RouterInit(testVirtualServiceYML, errorTestDestinationRuleYML)
-	//assert.Error(t, err)
-}
```

Comment:
Is this unit test still useful?

Reply:
Deprecated because Router Init has been refactored to RouteConfig, unit tests are in router_config_test#TestNewRouterConfigBuilder

Reply:
Ok, if the unit test was moved to other place, please remove them from code.